pkg/deploy/dashboard/deployment_dashboard.go

```diff
@@ -56,6 +56,12 @@ func (d *DashboardReconciler) getDashboardDeploymentSpec(ctx *deploy.DeployConte
 		corev1.EnvVar{
 			Name:  "CHE_URL",
 			Value: util.GetCheURL(ctx.CheCluster)},
+		corev1.EnvVar{
+			Name:  "CHE_NAMESPACE",
+			Value: ctx.CheCluster.Namespace},
+		corev1.EnvVar{
+			Name:  "CHE_CRD_OBJECT_NAME",
```

Comment:
Better name `CHECLUSTER_CR_NAME`.
If you decide to mount the env with a namspace name, then be consistent in naming. I suggest `CHECLUSTER_CR_NAMESPACE`.